packages/sdks/e2e/angular-ssr/src/app/catch-all.component.ts

```diff
@@ -48,6 +49,46 @@ export class CatchAllComponent {
       name: 'Hello',
       inputs: [],
     },
+    {
+      name: 'BuilderBlockWithClassName',
+      component: BuilderBlockWithClassNameComponent,
+      shouldReceiveBuilderProps: {
+        builderBlock: true,
+        builderContext: true,
+        builderComponents: true,
+      },
+      inputs: [
+        {
+          name: 'content',
+          type: 'uiBlocks',
+          defaultValue: [
+            {
+              '@type': '@builder.io/sdk:Element',
+              '@version': 2,
+              id: 'builder-c6e179528dee4e62b337cf3f85d6496f',
+              component: {
+                name: 'Text',
+                options: {
+                  text: 'Enter some text...',
+                },
+              },
+              responsiveStyles: {
+                large: {
+                  display: 'flex',
+                  flexDirection: 'column',
+                  position: 'relative',
+                  flexShrink: '0',
+                  boxSizing: 'border-box',
+                  marginTop: '20px',
+                  lineHeight: 'normal',
+                  height: 'auto',
+                },
+              },
+            },
+          ],
```

Comment:
could we move this as `DEFAULT_TEXT_ELEMENT` and export it from sdk-tests (see how getProps is used) and use it as the `defaultValue` everywhere?

Reply:
yeah, if you want to reduce all the repetition in each e2e test, you could add a `getBuilderBlockWithClassNameCustomComponentInfo` helper in https://github.com/BuilderIO/builder/blob/main/packages/sdks-tests/src/specs/index.ts

```ts
const getBuilderBlockWithClassNameCustomComponentInfo = (component) => ({
  component,
  name: 'BuilderBlockWithClassName',
  shouldReceiveBuilderProps: {
    builderBlock: true,
  },
 // ...
})
```

not required though, but would cleanup this PR a bunch